src/main.cpp

```diff
@@ -694,12 +735,12 @@ int main(int argc, char**argv)
   delete load_monitor; load_monitor = NULL;
   delete auth_store; auth_store = NULL;
   delete call_list_store; call_list_store = NULL;
-  delete m_store; m_store = NULL;
+  delete astaire_resolver; astaire_resolver = NULL;
```

Comment:
Yes, sorry - didn't spot that the code creating it had been moved rather than added.